### ISSUE TITLE
Style review notifications

### ISSUE THREAD

**Issue by a0eoc** [userstyles-world/userstyles.world#286]
Something like this:

> Somebody left a review on your userstyle

> ...
> 
> %Username% reviewed your userstyle %name% (with rating of %rating%|without a rating)( and provided a commentary).
> [Check it out].
> 
> ...

@vednoc WDYT?

**Comment by decembre** [comment]
+1
And add the possibility to respond to a review?

**Comment by Sceptrum** [comment]
> +1 And add the possibility to respond to a review?

Yes! I got a review (2/5) referred to an issue, that only exists on a different language, which I never applied my style onto.
This is pretty unfair. This is my first review after almost 2 years and now new faces will think it's bad eternally. I'm powerless and questioning myself, if I ever want to make updates again.
